src/lib/errors/authentication/wrong-password-error.js

```diff
@@ -0,0 +1,13 @@
+import HttpStatus from 'http-status-codes';
+import BaseHttpError from '../base-http-error';
+
+class WrongPasswordError extends BaseHttpError {
```

Comment:
Since there's no need to store a method on custom error prototype, you can and should avoid BaseHttpError and extend all of your custom errors direcly from default `Error`